### PR TITLE
Remove pretty_print from call to ansible_utils to_text

### LINKED ISSUE #225: Bug in tostring function in module_utils/network/junos/junos.py
##### SUMMARY
Getting an error due to Ansible util function to_text() called with an unexpected keywork argument

Looks due to faulty reworking in commit 2be892f96f1ae49a10bd65c28022c8fcc769d5d8 - 
The ansible [to_text](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/common/text/converters.py) function doesn't take pretty_print as a key work argument, this looks like an error pushing down pretty_print from tostring() args.

```
ansible_collections/junipernetworks/junos/plugins/module_utils/network/junos/junos.py\", line 92, in tostring\nTypeError: to_text() got an unexpected keyword argument 'pretty_print'\n"
```

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
module_utils/network/junos/junos.py

##### ANSIBLE VERSION

```paste below
ansible 2.10.5
  config file = /home/jreynolds/git/dcfabric/ansible.cfg
  configured module search path = ['/home/jreynolds/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /tmp/3.8/lib64/python3.8/site-packages/ansible
  executable location = /tmp/3.8/bin/ansible
  python version = 3.8.8 (default, Aug  2 2021, 14:57:02) [GCC 8.5.0 20210514 (Red Hat 8.5.0-3)]
```

##### COLLECTION VERSION

This was introduced between 2.3.0 and 2.4.0, and is still present in 2.5.0.

### PR DESCRIPTION
**Pull request by jake2184**
##### SUMMARY
Fixes #225 

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

plugins/module_utils/network/junos/junos.py

### CODE REVIEW

**Review by rohitthakur2590 — APPROVED** [review]

**Comment by rohitthakur2590** [comment]
recheck

File: changelogs/fix_tostring.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
```
Comment (rohitthakur2590) [inline changelogs/fix_tostring.yml:1]: - Could you please have fragment with indicator as `bugfixes` and move this file to  `changelogs/fragments` directory, example: https://github.com/ansible-collections/junipernetworks.junos/pull/293/files

**Review by rohitthakur2590 — CHANGES_REQUESTED** [review]

**Comment by avermeer-tc** [comment]
This was fixed by https://github.com/ansible-collections/junipernetworks.junos/pull/306

**Comment by rohitthakur2590** [comment]
fixed by https://github.com/ansible-collections/junipernetworks.junos/pull/306

**Pull request closed without merging**